services/mailgunService.js

```diff
@@ -0,0 +1,27 @@
+//const emailConfig = require('./email-config')();
+const emailConfig = require('../config/config');
+const mailgun = require('mailgun-js')(emailConfig.mailgun);
+
+
+
+module.exports.sendEmail = (recipient, message, attachment) =>
+  new Promise((resolve, reject) => {
+    const data = {
+    //  from: 'Gobinda Thakur <info@mg.gobindathakur.com>',
+      from: message.from,
+      to: recipient,
+      subject: message.subject,
+      text: message.text,
+      inline: attachment,
+      html: message.html,
```

Comment:
We discussed putting the template on the server side. Did you decide not to do that?